setup.py

```diff
@@ -37,8 +37,8 @@ def read_file(filename):
     setup_requires=['setuptools-git-version'],
     install_requires=[
         'dj-database-url>=0.4.2,<1',
-        'Django>=1.11,<1.12',
+        'Django>=1.11,<2.3',
         'django-haystack>=2.7,<2.9',
-        'django-localflavor>=1.1,<2',
+        'django-localflavor>=1.1,<4',
```

Comment:
I think we'll need to pin this to:
```suggestion
        'django-localflavor>=1.1,<3',
```
Because [django-localflavor 3.0 removed support for Django 1.11](https://github.com/django/django-localflavor/blob/3.0/setup.py#L123).

Reply:
Changed django-localflavor as requested